org.robomongo.Robo3T.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id": "org.robomongo.Robo3T",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "robo3t",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "robo3t",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -d -m 755 /app/bin",
+                "install -d -m 755 /app/include",
+                "install -d -m 755 /app/lib",
+
+                "cp -R bin/* /app/bin",
+                "cp -R include/* /app/include",
+                "cp -R lib/* /app/lib",
+
+                "install -D -m 644 icon.png /app/share/icons/hicolor/128x128/apps/org.robomongo.Robo3T.png",
+                "install -D -m 644 desktop.desktop /app/share/applications/org.robomongo.Robo3T.desktop",
+                "install -D -m 644 appdata.xml /app/share/appdata/org.robomongo.Robo3T.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download-test.robomongo.org/linux/robo3t-1.3.1-linux-x86_64-7419c406.tar.gz",
```

Comment:
this is an open source project, is there a reason you're not building from source?

Reply:
It requires building openssl and etc... (takes a lot of time to build)